src/proof/lrat/lrat_proof.h

```diff
@@ -52,6 +52,15 @@ class LratInstruction
    * @param out the stream to write to
    */
   virtual void outputAsText(std::ostream& out) const = 0;
```

Comment:
What tool can be used to process `out` after the operation of `outputAsText`, if any? Does it correspond to the format of the examples in the LRAT paper?

Reply:
Good question. The answer is the `ClausalBitVectorProof` (in review) and the `ProofManager`.

Since this method just writes an LFSC LRAT proof **value** (not a proof!), the `ClausalBitVectorProof` will write the axiom which uses a side condition to check the proof value and then prove bottom.

The `ProofManager` also needs to write a bunch of closing parentheses after that.

Unfortunately the answer is yes -- the proof manager (or some other component) write a bunch of closing parentheses to the stream.